### PR TITLE
[Message Center] Add Material Design toast and snackbar message center

### PR DESCRIPTION
**Pull request by Tommass**
## Description

We develop a brand new message center, respecting Material Design specifications.
It is based on Material Design Lite HTML and CSS Templating.

For more information about MDL version, please read http://www.getmdl.io/components/index.html#snackbar-section

For more information about MD specification, please read https://www.google.com/design/spec/components/snackbars-toasts.html#snackbars-toasts-specs

## Code

[Add code]

## Dependencies

This PR will upgrade Material Design Lite version.

### CODE REVIEW

File: src/components/message-center/style/message-center.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+
+[data-focus="snackbar-message-center"] {
+
```
Comment (pierr) [inline src/components/message-center/style/message-center.scss]: Useless?

**Comment by pierr** [comment]
Great update of the message center 
Just two questions:
- don't we need to maintain an old version of it for the projects?
- I would just have done two PR one for material and one for this feature.

**Comment by pierr** [comment]
Thanks for the unit tests

**Comment by pierr** [comment]
The result is great!
I just maybe would have split the component with a sate aware component using something such as the connect behaviour and a pure function component to be less dependent to the flux stores implementations. 
This is not necessary for the merge and can be done after. 
What do you think?

File: src/components/message-center/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+            after(() => {
+                unmountComponentAtNode(findDOMNode(reactComponent).parentNode);
+            });
+            before(() => {
+                const MessageCenterComponent = <MessageCenter />;
+                reactComponent = renderIntoDocument(MessageCenterComponent);
+                domNode = findDOMNode(reactComponent);
+            });
+            before(() => {
+                addSuccessMessage(message);
```
Comment (pierr) [inline src/components/message-center/__tests__/index.js]: Put it in the same before block

File: src/components/message-center/style/message-center.scss (hunk 1 above)
Comment (Tommass) [inline src/components/message-center/style/message-center.scss]: Juste to allow a develop to know what he can overload. Do i remove it ? (we have done it on others components)

File: src/components/message-center/__tests__/index.js (hunk 2 above)
Comment (Tommass) [inline src/components/message-center/__tests__/index.js]: we mean the 2 before ?
if i do it does not work. i have a "hook" issue.

**Pull request merged**